### PR TITLE
tests: Disable driver check

### PR DESCRIPTION
**Pull request by ncesario-lunarg**
Disable the IsDriver check if the VK_LAYER_TESTS_SKIP_PLATFORM_CHECK
environment variable is set.

### CODE REVIEW

File: tests/vkrenderframework.cpp [hunk 1]
```diff
@@ -685,21 +685,30 @@ void VkRenderFramework::GetPhysicalDeviceFeatures(VkPhysicalDeviceFeatures *feat
     }
 }
 
-bool VkRenderFramework::IsPlatform(PlatformType platform) {
+// static
+bool VkRenderFramework::SkipDisabled() {
```
Comment (lunarpapillo) [inline tests/vkrenderframework.cpp:689]: I'm sort of surprised this doesn't have to be marked `static` like it is in the header?  Maybe I need to work more on my modern C++...

**Review by lunarpapillo — APPROVED** [review]
The code and functionality look great.  But the name of the variable `VK_LAYER_TESTS_SKIP_PLATFORM_CHECK` is now misleading.  The natural replacement `VK_LAYER_TESTS_SKIP_DISABLED` is clear from the code but may also be misinterpreted by someone not familiar with the code (who may conclude "of course we want to skip disabled tests!").

There must be a clear answer, but nothing's coming to me right now...

File: tests/vkrenderframework.cpp (hunk 1 above)
Comment (ncesario-lunarg) [reply tests/vkrenderframework.cpp:689]: Yes, this is an oddity (from my perspective, at least), which is why I usually add a `static` comment to remind myself.

**Comment by ncesario-lunarg** [comment]
> But the name of the variable VK_LAYER_TESTS_SKIP_PLATFORM_CHECK is now misleading

I agree. I made this change pretty hastily. I'll update the env var name, as I think internal CI is the only one using it so far.

**Comment by ncesario-lunarg** [comment]
I went with `VK_LAYER_TESTS_IGNORE_DISABLE_CHECKS` for now...

**Review by Tony-LunarG — APPROVED** [review]
LGTM

**Pull request merged**